crates/nu-command/src/filesystem/save.rs

```diff
@@ -53,6 +53,8 @@ impl Command for Save {
             .switch("append", "append input to the end of the file", Some('a'))
             .switch("force", "overwrite the destination", Some('f'))
             .switch("progress", "enable progress bar", Some('p'))
+            .switch("out-append", "only for internal use", None)
+            .switch("err-append", "only for internal use", None)
```

Comment:
Why are these two switches "only for internal use"?

Reply:
We needs these two switches to indicates if we want to use append mode for stdout and stderr separately.

But I don't think user don't needs these flags, consider the following example:
```nushell
echo aaa | save -a b.txt
```
A simple `-a` flag is enough for use.  We can't pipe stdout and stderr command to next command separately

Reply:
can't we just hide these two switches?
i have to admit it looks a bit confusing from the user's point of view :confused:

Reply:
I'd just leave them and change "only for internal use" to something more descriptive. Then, in the extra_usage mention something about these parameters.

Reply:
Thanks for your suggestion!  Would you please check the updated help message?

Reply:
Thanks @WindSoilder for typing something more user friendly. However, I'm worried that it's still pretty confusing. What will the user think when they see `--append`, `--out-append` and `--err-append`; how will they know which one to choose?

Since (I'm guessing) `--out-append` and `--err-append` are only required by your new append operator, I wonder if there's a better way to pass these and not expose them to users. For example, are these something that could be in the stack or engine_state?

Reply:
Yeah, we can make it into parser_info, so `--out-append` and `--err-append` won't show to user

Reply:
Thanks!